engine-api/src/schema/block.rs

```diff
@@ -0,0 +1,93 @@
+use {
+    moved::{
+        primitives::{Address, Bytes, B2048, B256},
+        types::state::{BlockResponse, TransactionResponse},
+    },
+    serde::{Deserialize, Serialize},
+};
+
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct GetBlockResponse {
+    /// the block number. null when its pending block.
```

Comment:
nit: We usually capitalize the comments, ie `/// The ...`
Applies to all of them below.